tests/api/test_bookmarks.py

```diff
@@ -1,12 +1,14 @@
 import requests
 import json
+from flask import jsonify
 import uuid
 
 
 def test_get_empty_bookmark(flask_api_url):
-    payload = {'email': 'admin@knex.com', 'password': "admin"}
-    response = requests.post(flask_api_url + "/api/users/login",
-                             data=payload)
+    response = requests.post(flask_api_url + '/api/users/login',
```

Comment:
this is pointless, you need to get a session first
session = requests.Session()
session.posrt(login)